src/components/Comments/ContainerComment.js

```diff
@@ -95,6 +95,7 @@ const ContainerComment = ({
             {item.user ? (
               <ListComment
                 key={`p${index}`}
+                feedId={feedId}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `feedId` as a prop to the `ListComment` component seems appropriate. It's important to ensure that this new prop is being correctly passed down from parent components and used appropriately within the `ListComment` component.
```diff
+                feedId={feedId}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->